stage1/init/common/mount.go

```diff
@@ -45,13 +56,28 @@ func IsMountReadOnly(vol types.Volume, mountPoints []types.MountPoint) bool {
 	return isMPReadOnly(mountPoints, vol.Name)
 }
 
-// GenerateMounts maps MountPoint paths to volumes, returning a list of Mounts.
-func GenerateMounts(ra *schema.RuntimeApp, volumes map[types.ACName]types.Volume) []schema.Mount {
+func convertedFromDocker(ra *schema.RuntimeApp) bool {
+	ann := ra.Annotations
+	_, ok := ann.Get("appc.io/docker/repository")
+	return ok
+}
+
+// GenerateMounts maps MountPoint paths to volumes, returning a list of mounts,
+// each with a parameter indicating if it's an implicit empty volume from a
+// Docker image.
+func GenerateMounts(ra *schema.RuntimeApp, volumes map[types.ACName]types.Volume) []mountWrapper {
```

Comment:
Hm, probably because before refactoring this was used by some kvm-specific code, which probably was in a separate package.